src/SdkGenerator/SdkGenerator.csproj

```diff
@@ -1,74 +1,85 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <OutputType>Exe</OutputType>
-        <TargetFramework>net7.0</TargetFramework>
-        <PackAsTool>true</PackAsTool>
-        <ToolCommandName>SdkGenerator</ToolCommandName>
-        <PackageOutputPath>../../</PackageOutputPath>
-    </PropertyGroup>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>SdkGenerator</ToolCommandName>
+    <PackageReadmeFile>docs/README.md</PackageReadmeFile>
+    <Description>A command line dotnet tool to retrieve swagger (OpenAPI) files and generate
+      easy-to-use markdown documentation or friendly software development kit libraries. Can be
+      automated using Github Workflows.</Description>
+    <PackageTags>SDK generator swagger openapi swashbuckle</PackageTags>
+    <Copyright>Copyright 2021 - 2023</Copyright>
+    <PackageReleaseNotes># 1.1.6
+      August 21, 2023
+
+      * Fixed capitalization of DotnetToolSettings.xml
+    </PackageReleaseNotes>
+    <PackageIcon>docs/icons-puzzle.png</PackageIcon>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Version>1.1.6</Version>
+    <Authors>Ted Spence</Authors>
+    <!-- Project Url is filled in by sourcelink in the .NET 8 SDK, but you can add it explicitly via
+    package -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
This tells sourcelink to fill in the repository data from git automatically, including commit SHA information.